polar_training2tcx

```diff
@@ -173,13 +173,13 @@ def output_tcx(parsed)
                           }
                         }
                       end
-                    }.xmlns = 'http://www.garmin.com/xmlschemas/TrainingCenterDatabase/v2'
+                    }
 
                     i += 1
                     elapsed += recording_interval
                     elapsed_with_pauses += recording_interval
                   end
-                }.xmlns = 'http://www.garmin.com/xmlschemas/TrainingCenterDatabase/v2'
```

Comment:
Same here, Polar Flow does include the namespace for all <Track>, except the first one. Not sure why, maybe to please Strava too?
Those namespaces may not be needed for usual use cases, we can try to remove them and see if someone needs them.

Reply:
Maybe it depends on the version and/or the platform of the PolarFlow app. I used their Windows applicaiton to load the trainings, and then downloaded the results via a browser.